README.md

```diff
@@ -41,4 +45,5 @@ import AutoTypingText from 'react-native-auto-typing-text';
 |**`charMovingTime`**|`number`|time to type each character|
 |**`delay`**|`number`|Delay time before typing|
 |**`style`**|`string`|Style for text|
+|**`containerStyle`**|`string`|Style for text container|
```

Comment:
`containerStyle` must be `object`

Reply:
I thought that may more sense but tried to match the previous, i'll make the change now

Reply:
Yep, I thought u copy pasted as it was same 😂

Reply:
While you are at it, change [this](https://github.com/JackThomson2/react-native-auto-typing-text/blob/ae7274a46793a059cbe9ca8e00e2b7ff7a322533/AutotypingText.js#L13) too. It should be a `number` not `string`

Reply:
Oops sorry didn't see that I'll add another commit